utils/storage-type-parser.ts

```diff
@@ -0,0 +1,18 @@
+import { DBClusterStorageType } from "aws-cdk-lib/aws-rds";
+import { StorageType } from "../lib/AwsAuroraServerlessStackProps";
+
+export const parseStorageType = (storageType: StorageType): DBClusterStorageType => {
+    return storageType === StorageType.AURORA_IOPT1 ? DBClusterStorageType.AURORA_IOPT1 : DBClusterStorageType.AURORA;
+}
+
+export const parseStorageTypeFromEnv = (): StorageType => {
+    const storageType = process.env.STORAGE_TYPE;
+    if (!storageType) {
+        throw new Error('STORAGE_TYPE is not set');
+    }
+    const acceptedValues = [StorageType.AURORA, StorageType.AURORA_IOPT1];
+    if (!acceptedValues.includes(storageType as StorageType)) {
+        throw new Error(`Invalid STORAGE_TYPE value: ${storageType}`);
```

Comment:
**Suggestion:** Consider using uppercase for the storage type validation to ensure case-insensitive comparison. This makes the configuration more user-friendly by accepting both uppercase and lowercase values.
> 🎇  **Medium Priority** 
 >[Category: **`Enhancement`**, Importance: **`7`**]
> **Reason:** This suggestion improves user experience by allowing case-insensitive input for storage type validation, which can reduce configuration errors. The enhanced error message also provides clarity. However, the improvement is not critical to functionality, so it receives a moderate score.

```suggestion
    const storageTypeUpper = storageType.toUpperCase();
    const acceptedValues = [StorageType.AURORA, StorageType.AURORA_IOPT1];
    if (!acceptedValues.includes(storageTypeUpper as StorageType)) {
        throw new Error(`Invalid STORAGE_TYPE value: ${storageType}. Must be one of: ${acceptedValues.join(', ')}`);
    }
```
- [ ]  **I have reviewed this code suggestion and addressed the relevant ones.**